tests/test_unit.py

```diff
@@ -143,11 +143,13 @@ def constructor():
             parameter = constructor()
 
             assert parameter.value == outcome
-            assert parameter.get_value_in_default_format() == outcome_default_format
+            assert parameter.get_value_in_default_units() == outcome_default_format
 
             # setting value during request
             assert parameter.set_par(input_value) == outcome_default_format
-            assert parameter.value == outcome
+            assert parameter.value == outcome_default_format
```

Comment:
So then get_value_in_default_units() becomes redundant. Previous behaviour was totally logical for me.

Reply:
I am adjusting also this.